src/components/PageHeadingVariables.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <page-heading class="page-heading-variables" :crumbs="crumbs">
+    <template v-if="can.create.variable" #after-crumbs>
+      <p-button v-if="can.create.work_pool" inset size="xs" icon="PlusIcon" @click="open" />
```

Comment:
I'm guessing this permission check isn't supposed to be here

Reply:
Oops yup - fixed